rllib/utils/replay_buffers/reservoir_buffer.py

```diff
@@ -61,7 +61,14 @@ def add(self, batch: SampleBatchType, **kwargs) -> None:
                 self._storage[idx] = batch
 
                 assert batch.count > 0, batch
-                warn_replay_capacity(item=batch, num_items=self.capacity / batch.count)
+                warn_replay_capacity(item=batch,
+                                     num_items=self.capacity / batch.count)
+
+    @ExperimentalAPI
+    @override(ReplayBuffer)
+    def sample(self, num_items: int, **kwargs) -> Optional[SampleBatchType]:
+        """Samples `num_items` timesteps from this buffer."""
+        raise NotImplementedError
```

Comment:
is there anything special about sampling from a reservoir buffer?
why not simply ReplayBuffer.sample(...)?